Presentation.Web/Tests/01_Parallel/data-processing/tabs/data-processing-registration.main.e2e.spec.ts

```diff
@@ -30,19 +33,18 @@ describe("Data processing registration main detail tests", () => {
     var dropdownYes = "Ja";
 
     var today = GetDateHelper.getTodayAsString();
-
+    
     beforeAll(() => {
-        loginHelper.loginAsLocalAdmin();
+        loginHelper.loginAsGlobalAdmin();
```

Comment:
In stead of doing this.. in the test, login as global admin, create the organization, then login as local admin again and run the remainder of the test.

The reason we must not run tests as global admin for the regular user's use cases (creating organizations is a global admin use case) is that global admin can do anything and is not assigned to more than a select few.